python/perf-kernels/rmsnorm.py

```diff
@@ -16,6 +16,14 @@ def is_hip():
     return triton.runtime.driver.active.get_current_target().backend == "hip"
 
 
+if torch.cuda.is_available():
```

Comment:
I don't think this conditional makes sense. If CUDA isn't available, we don't really care about triton anyways. So this is a nop.

cuda is considered available for Triton so we can skip it. I'd just set NUM_SMS to equal the multi_processor_count.

Reply:
actually below three lines works for both CUDA and HIP backend.

```
current_device_index = torch.cuda.current_device()
current_device = torch.cuda.get_device_properties(current_device_index)
NUM_SMS = current_device.multi_processor_count
```

Reply:
done